docs/site/Appsody-LoopBack.md

```diff
@@ -0,0 +1,990 @@
+---
+lang: en
+title: 'Deploying to Cloud with Appsody'
+keywords: LoopBack 4.0, LoopBack 4, Appsody
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Appsody-LoopBack.html
+---
+
+## LoopBack 4 & Appsody
+
+[Appsody](https://appsody.dev/docs#welcome-to-appsody) is designed to help
+develop containerized applications for the cloud.
+
```

Comment:
Should we mention https://kabanero.io/?

Reply:
> Should we mention https://kabanero.io/?

@raymondfeng , how do we want to mention it? I downloaded and used the appsody CLI with homebrew. I didn't use kabanero.

Reply:
My initial thought was to make our users aware that Kabanero is a broader offering that uses Appsody for cloud native app dev/dep.